src/metabase/api/dashboard.clj

```diff
@@ -502,6 +504,41 @@
       (u/prog1 (first (last-edit/with-last-edit-info [dashboard] :dashboard))
         (events/publish-event! :event/dashboard-read {:object-id (:id dashboard) :user-id api/*current-user-id*})))))
 
+(api/defendpoint GET "/:id/items"
+  "Get Dashboard with ID."
+  [id]
+  {id ms/PositiveInt}
+  (api/read-check :model/Dashboard id)
+  ;; query copied from metabase.api.collection to match the shape of api/collection/<:id|root>/items
+  (let [query      {:select    (cond->
+                                [:c.id :c.name :c.description :c.entity_id :c.collection_position :c.display :c.collection_preview
+                                 :last_used_at :c.collection_id :c.archived_directly :c.archived :c.dataset_query :c.database_id
+                                 [(h2x/literal "card")  :model]
+                                 [{:select   [:status]
+                                   :from     [:moderation_review]
+                                   :where    [:and
+                                              [:= :moderated_item_type "card"]
+                                              [:= :moderated_item_id :c.id]
+                                              [:= :most_recent true]]
+                                      ;; limit 1 to ensure that there is only one result but this invariant should hold true, just
+                                      ;; protecting against potential bugs
```

Comment:
would be clearer to put this comment below the `:order-by` (and next to the `:limit`).